test/dotnet-core-uninstall.Tests/Shared/Configs/CommandLineConfigsTests.cs

```diff
@@ -384,7 +434,32 @@ internal void TestGetUninstallMainOptionVersionBeforeOptionException(string opti
         [InlineData("--sdk --runtime --aspnet-runtime", BundleType.Sdk | BundleType.Runtime | BundleType.AspNetRuntime)]
         [InlineData("--hosting-bundle --aspnet-runtime", BundleType.AspNetRuntime | BundleType.HostingBundle)]
         [InlineData("--hosting-bundle --sdk --all", BundleType.Sdk | BundleType.HostingBundle)]
-        internal void TestGetTypeSelectionRootCommand(string command, BundleType expected)
+        internal void TestGetTypeSelectionRootCommandWindows(string command, BundleType expected)
+        {
+            var parseResult = CommandLineConfigs.UninstallRootCommand.Parse(command);
+
+            parseResult.Errors.Should().BeEmpty();
+            parseResult.UnparsedTokens.Should().BeEmpty();
+            parseResult.UnmatchedTokens.Should().BeEmpty();
+
+            parseResult.RootCommandResult.GetTypeSelection()
+                .Should().Be(expected);
+        }
+
+        [MacOsOnlyTheory]
+        [InlineData("", BundleType.Sdk | BundleType.Runtime)]
+        [InlineData("--sdk", BundleType.Sdk)]
+        [InlineData("--runtime", BundleType.Runtime)]
+        [InlineData("--sdk --runtime", BundleType.Sdk | BundleType.Runtime)]
+        [InlineData("-v q", BundleType.Sdk | BundleType.Runtime)]
+        [InlineData("--sdk --verbosity minimal", BundleType.Sdk)]
+        [InlineData("-v normal --runtime", BundleType.Runtime)]
+        [InlineData("--sdk --verbosity diag --runtime", BundleType.Sdk | BundleType.Runtime)]
+        [InlineData("--all", BundleType.Sdk | BundleType.Runtime)]
+        [InlineData("--sdk --all-but 2.2.300 2.1.700", BundleType.Sdk)]
+        [InlineData("--runtime --all-below 3.0.1-preview-10086", BundleType.Runtime)]
+        [InlineData("--sdk --runtime --all-previews", BundleType.Sdk | BundleType.Runtime)]
```

Comment:
suggestion, add "`--aspnet-runtime` will throw in macos" tests

Reply:
Thanks for your suggestion! It's been added.